src/components/more-from-xl-blog/more-from-xl-blog.module.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../css/colors.scss';
+
+.moreFromXlContainer {
+  background-color: $blueThree;
+}
+
+.titleStyle {
+  text-decoration: none;
+}
+
+.titleContainer {
+  margin: 0 0 6rem 8.9rem;
+  padding-top: 6rem;
+}
+
+.blogsContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-items: center;  
+}
+
+.cardStyles {
+  width: 23%;
```

Comment:
Why do you not use a grid o similar?

Reply:
I don't know, I made the component with that form, but, I can change it for grid.

Reply:
Do you prefer that?